test/test_quantized_nn_mods.py

```diff
@@ -65,7 +65,7 @@ def test_conv_api(self, use_bias):
 
         b = torch.randn(oC, dtype=torch.float32) if use_bias else None
         q_bias = torch.quantize_linear(b, scale=1.0 / 1024, zero_point=0, dtype=torch.qint32) if use_bias else None
-        q_filters_ref = torch.ops.quantized.conv_prepack(qw.permute([0, 2, 3, 1]),
```

Comment:
Don't you also need similar changes to the tests in test_quantized.py?

Reply:
oh, I think I put it accidentally in https://github.com/pytorch/pytorch/pull/26242 - let me just squash them together